cartridges/int_algolia/cartridge/scripts/algolia/steps/algoliaProductDeltaIndex.js

```diff
@@ -140,6 +140,7 @@ exports.beforeStep = function(parameters, stepExecution) {
 
     // return OK if the folder doesn't exist, this means that the CatalogDeltaExport job step finished OK but didn't have any output (there were no changes)
     if (!l0_deltaExportDir.exists()) {
+        logger.info('Export directory does not exist (CatalogDeltaExport didn\'t generate anything here): ' + l0_deltaExportDir.getFullPath());
```

Comment:
This should be more explicit and it should be an error. The `deltaExportJobName` parameter should not be changed on the job step level and it needs to be left to be overwritten by the job-level parameter since both job steps need to have access to the same value (so that the second step finds the export zips generated by the delta export step). I called attention to this in the documentation, so we should treat it as an error if clients change this value on the step level.

Reply:
Let's leave it like this, the documentation warns that the value should not be changed.